README.md

```diff
@@ -1,108 +1,97 @@
 # __StonkScraper__
-This API is a WIP. Check back often for new features!<br>
 [See a live implementation here!](https://boringstocks.live/)
 
-`StonkScraper` is a lightweight API that scrapes Yahoo's financial pages for a defined stock index. It is minimal by design but returns most of the data that the average daytrader would require to decide whether to buy or sell GME.
+`StonkScraper` is a lightweight API built off the robin_stock API.
 
 ## Usage
 
 Primary endpoint: `http://api.boringstocks.live/`
 
-### Current Index Data
-Request `https://api.boringstocks.live/v1/<stock_index>`<br>
-`StonkScraper` returns a JSON containing these data points:
-  - Current price
-  - Range of historical data (not including the current day)
-  - Market cap
-  - Market status (open/close)
-  - Opening price
-  - Price change (points & percent)
-  - Day's range (date, high, low, & close)
-  - Index
-  - Timestamp
-  - Average volume
-  - Volume
+### Index Data
+Request `https://api.boringstocks.live/v2/<stock_index>`<br>
+`StonkScraper` returns a JSON containing:
```

Comment:
Maybe you could move this above ^^ to the Usage header, since it applies to all routes